druid/src/lib.rs

```diff
@@ -127,7 +127,7 @@
 //! [`image` crate]: https://crates.io/crates/image
 
 #![deny(
-    broken_intra_doc_links,
+    rustdoc::broken_intra_doc_links,
```

Comment:
does not compile on rustc 1.51

Reply:
Is this an issue? It does work on 1.52, and that's already two versions behind current..

Reply:
:shrug: I am not sure about bumping msrv just for this.

Reply:
no sweat, happy to just merge the other one